docs/source/reference/storage.rst

```diff
@@ -63,8 +63,8 @@ When specifying a storage object, you can specify either of two modes:
     they are accessed by applications. This mode also allows applications to
     write to the mount path. All writes are replicated to remote bucket (and
     any other VMs mounting the same bucket). Please note that this mode
-    uses a close-to-open consistency model, which means a file write is
-    committed to the backing store only after :code:`close()` is called on it.
+    uses a close-to-open consistency model. This means calling :code:`close()`
+    on a file will guarantee that any subsequent reads will see the latest data.
```

Comment:
“close-to-open” can be unfamiliar. Maybe we can add a sentence using the user’s own words, along the lines of “every close will trigger an re-upload of the entire file (true?) to the backing bucket”.

Reply:
Thanks! I added this but it was making the bullet point too long, so I moved these details to two notes after the example. PTAL